packages/core/src/internal/clients/cache/AbstractCacheClient.ts

```diff
@@ -209,6 +212,46 @@ export abstract class AbstractCacheClient implements ICacheClient {
     return await client.delete(cacheName, key);
   }
 
+  /**
+   * Gets the value stored for the given keys.
+   *
+   * @param {string} cacheName - The cache to perform the lookup in.
+   * @param {string[] | Uint8Array[]} keys - The list of keys to look up.
+   * @returns {Promise<GetBatch.Response>} -
+   * {@link GetBatch.Success} containing the values if they were found.
+   * {@link GetBatch.Error} on failure.
+   */
+  public async getBatch(
+    cacheName: string,
+    keys: string[] | Uint8Array[]
```

Comment:
I think that theoretically this could support `List<string | Uint8Array>` (mixed key types).

I don't think it's super important for us to support this but if it's easy it'd be nice.